src/main/java/pl/betoncraft/betonquest/config/Config.java

```diff
@@ -110,24 +114,42 @@ public Config(final boolean verboose) {
      * Creates package with the given name and populates it with default quest
      *
      * @param packName name of the new package
+     * @param populate if the files should be populated with the example quest or left empty
      * @return true if the package was created, false if it already existed
      */
     @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE")
-    public static boolean createDefaultPackage(final String packName) {
+    public static boolean createDefaultPackage(final String packName, final boolean populate) {
         final File def = new File(instance.root, packName.replace("-", File.separator));
         if (!def.exists()) {
             LogUtils.getLogger().log(Level.INFO, "Deploying " + packName + " package!");
             def.mkdirs();
-            saveResource(def, "default/main.yml", "main.yml");
-            saveResource(def, "default/events.yml", "events.yml");
-            saveResource(def, "default/conditions.yml", "conditions.yml");
-            saveResource(def, "default/journal.yml", "journal.yml");
-            saveResource(def, "default/items.yml", "items.yml");
-            saveResource(def, "default/objectives.yml", "objectives.yml");
-            saveResource(def, "default/custom.yml", "custom.yml");
+            if (populate) {
+                saveResource(def, "default/main.yml", "main.yml");
+                saveResource(def, "default/events.yml", "events.yml");
+                saveResource(def, "default/conditions.yml", "conditions.yml");
+                saveResource(def, "default/journal.yml", "journal.yml");
+                saveResource(def, "default/items.yml", "items.yml");
+                saveResource(def, "default/objectives.yml", "objectives.yml");
+                saveResource(def, "default/custom.yml", "custom.yml");
+            } else {
+                try {
+                    new File(def, "main.yml").createNewFile();
+                    new File(def, "events.yml").createNewFile();
+                    new File(def, "conditions.yml").createNewFile();
+                    new File(def, "journal.yml").createNewFile();
+                    new File(def, "items.yml").createNewFile();
+                    new File(def, "objectives.yml").createNewFile();
+                    new File(def, "custom.yml").createNewFile();
+                } catch (IOException e) {
+                    LogUtils.getLogger().log(Level.WARNING, "Could not create file: " + e.getMessage());
+                    LogUtils.logThrowable(e);
+                }
+            }
             final File conversations = new File(def, "conversations");
             conversations.mkdir();
-            saveResource(conversations, "default/conversations/innkeeper.yml", "innkeeper.yml");
+            if (populate) {
+                saveResource(conversations, "default/conversations/innkeeper.yml", "innkeeper.yml");
+            }
```

Comment:
How about moving this part into the above populate block by creating the conversation directory before the first `if (populate)` block?